components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -348,7 +350,7 @@ lnd/clm2/surfdata_map/surfdata_twpx4v1_simyr2000_c170706.nc</fsurdat>
 <flanduse_timeseries hgrid="ne30np4"       sim_year_range="1850-2000" 
  use_crop=".false."  >lnd/clm2/surfdata_map/landuse.timeseries_ne30np4_hist_simyr1850-2015_c180306.nc</flanduse_timeseries>
  <flanduse_timeseries hgrid="ne30np4.pg2"  sim_year_range="1850-2000" 
- use_crop=".false."  >lnd/clm2/surfdata_map/landuse.timeseries_ne30np4_hist_simyr1850-2015_c180306.nc</flanduse_timeseries>
+ use_crop=".false."  >DUMMY</flanduse_timeseries>
```

Comment:
Why did you have to make this change to the ne30np4.pg2 CLM code?

Reply:
I think I had to put something there for it to run, but for the F-compset runs it doesnt use it anyway. Since I had put the ne30np4 file there as a placeholder I thought it would be better to put "DUMMY" so it would be clear where the problem was if someone tried to run a transient simulation with the land model on the pg2 grid.